core/src/main/java/com/linecorp/armeria/server/AnnotatedHttpServiceConfiguratorSetters.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server;
+
+import static java.util.Objects.requireNonNull;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableList.Builder;
+
+import com.linecorp.armeria.server.annotation.ExceptionHandlerFunction;
+import com.linecorp.armeria.server.annotation.RequestConverterFunction;
+import com.linecorp.armeria.server.annotation.ResponseConverterFunction;
+
+/**
+ * A setter class for binding to {@link AnnotatedHttpServiceConfigurator}.
+ */
+public final class AnnotatedHttpServiceConfiguratorSetters {
+
+    private final Builder<ExceptionHandlerFunction> exceptionHandlers = ImmutableList.builder();
+    private final Builder<RequestConverterFunction> requestConverters = ImmutableList.builder();
+    private final Builder<ResponseConverterFunction> responseConverters = ImmutableList.builder();
+
+    /**
+     * Configures the specified {@link ExceptionHandlerFunction}s with the annotated service.
+     */
+    public void configureExceptionHandlers(Iterable<ExceptionHandlerFunction> exceptionHandlers) {
```

Comment:
How about returning `AnnotatedHttpServiceConfiguratorSetters` so that a user can set the handlers fluently?